samples/subsys/logging/logger/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _logger_sample:
+
+Logging
+###########
+
+Overview
+********
+A simple application that demonstrates use of logging subsystem. It demonstrates
```

Comment:
Change to:

    This is a sample application demonstrating use of the logging
    subsystem main features: severity levels, time stamping, module
    level filtering, and instance level filtering.  It also showcases
    logging bandwidth performance.

Reply:
Done

Reply:
I don't see the changes submitted yet :(